plugins/chunter-impl/src/components/Message.svelte

```diff
@@ -49,6 +53,20 @@
     return (await client.findAll(core.class.Account, { _id: userId }))[0]
   }
 
+  let query: QueryUpdater<Notification> | undefined
+
+  query = client.query(
```

Comment:
I think we need to try not call any query per individual messages, if we had it already we need to try get rid of it.